frontend/src/App.css

```diff
@@ -1,42 +1,81 @@
-#root {
-  max-width: 1280px;
-  margin: 0 auto;
-  padding: 2rem;
-  text-align: center;
-}
+:root {
+  font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
+  line-height: 1.5;
+  font-weight: 400;
+
+  color-scheme: light dark;
+  color: #ffffff;
+  background-color: #1b361c; /* Zielony, ciemny kolor */
+
+  font-synthesis: none;
+  text-rendering: optimizeLegibility;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
 
-.logo {
-  height: 6em;
-  padding: 1.5em;
-  will-change: filter;
-  transition: filter 300ms;
+  --main-green: #4caf50; /* Główny zielony kolor */
+  --hover-green: #388e3c; /* Ciemniejszy zielony do hover */
+  --background-light: #e8f5e9; /* Jasne tło dla light mode */
 }
-.logo:hover {
-  filter: drop-shadow(0 0 2em #646cffaa);
+
+a {
+  font-weight: 500;
+  color: var(--main-green);
+  text-decoration: inherit;
 }
-.logo.react:hover {
-  filter: drop-shadow(0 0 2em #61dafbaa);
+a:hover {
+  color: var(--hover-green);
 }
 
-@keyframes logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-width: 320px;
+  min-height: 100vh;
+  background-color: var(--background-light);
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  a:nth-of-type(2) .logo {
-    animation: logo-spin infinite 20s linear;
-  }
+h1 {
+  font-size: 3.2em;
+  line-height: 1.1;
+  color: var(--main-green);
+  text-align: center;
 }
 
-.card {
-  padding: 2em;
+button {
+  border-radius: 8px;
+  border: 1px solid var(--main-green);
+  padding: 0.6em 1.2em;
+  font-size: 1em;
+  font-weight: 500;
```

Comment:
are we sure that we want to apply irt to every button? please use className instead this will be safer